### PR TITLE
Add September 2019 slide deck

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

File: 2019-September.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <p>Hosted by</p>
+                            <img id="hoster-small" class="hoster" src="" alt=""">
+                        </div>
+
+                        <div>
+                            <ul class="pointless">
+                                <li>18h00 - 18h30: Welcome</li>
+                                <li>18h30 - 19h30: Dinner</li>
```
Comment (lcobucci) [inline 2019-September.html]: Will the dinner be before of the talk again?

File: 2019-September.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+                <!-- <section data-background="img/pagination_promo.jpg">
+                </section> -->
+
+                <section>
+                    <p>
+                        <img src="img/checkin-to-win.png" alt="Check-in to win"/>
+                    </p>
+
+                    <a href="http://raffles.amsterdamphp.nl">http://raffles.amsterdamphp.nl</a>
```
Comment (lcobucci) [inline 2019-September.html]: Maybe remove this link since raffler isn't working?

File: 2019-September.html (hunk 1 above)
Comment (carusogabriel) [reply 2019-September.html]: @WyriHaximus Do you mind change it?

File: 2019-September.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+                    <h4>Until end of July</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Raffles</h1>
+                    <p style="margin-top: 3em">
+                        <img src="img/checkin-to-win.png" alt="Check-in to win"/>
+                    </p>
+
+                    <a href="http://raffles.amsterdamphp.nl">http://raffles.amsterdamphp.nl</a>
```
Comment (lcobucci) [inline 2019-September.html]: Here too

Comment (WyriHaximus) [reply 2019-September.html]: Nuked it :fire:

File: 2019-September.html (hunk 2 above)
Comment (WyriHaximus) [reply 2019-September.html]: Nuked it :fire:

File: 2019-September.html (hunk 1 above)
Comment (WyriHaximus) [reply 2019-September.html]: Removed dinner

**Pull request merged**